environment-api/build.gradle

```diff
@@ -32,9 +32,10 @@ dependencies {
   implementation     group: 'com.sun.xml.ws',                name: 'jaxws-rt',                       version: "2.3.0"
 
   testImplementation group: 'org.junit.jupiter',             name: 'junit-jupiter-api',              version: junitJupiterVersion
+  testImplementation group: 'org.junit.jupiter',             name: 'junit-jupiter-params',           version: junitJupiterVersion
   testImplementation group: 'com.openpojo',                  name: 'openpojo',                       version: openPojoVersion
   testImplementation group: 'org.ow2.asm',                   name: 'asm',                            version: asmVersion
-
+  testImplementation group: 'org.hibernate',                 name: 'hibernate-validator',            version: hibernateValidatorVersion
```

Comment:
Why do you add to test only?

Reply:
Because it is used only for testing validation in special cases.
The Environment module uses Spring Boot and Jersey which in turn internally includes the Hibernate validator (see `./gradlew :environment:dependencies`)